except_star.md

```diff
@@ -1,33 +1,33 @@
-### Disclaimer
+# Introducing try..except* syntax
 
-* I'm going to be using the `ExceptionGroup` name in this issue, even
-  though there are other alternatives, e.g. `AggregateException`. Naming of the
-  "exception group" object is outside of the scope of this issue.
+## Disclaimer
 
-* This issue is primarily focused on discussing the new syntax modification
-  proposal for the `try..except` construct, shortly called "except*".
+* We use the `ExceptionGroup` name herein, even though there
+  are other alternatives, e.g. `AggregateException`. Naming of the
```

Comment:
And `MultiError`?

Reply:
Yeah, I'll add it. But I think that if we decide to transform this document into our PEP we'll be removing parts of this disclaimer, i.e. in this case we actually will discuss the naming in the PEP.